src/Digdir.Domain.Dialogporten.Application/Externals/Presentation/IUser.cs

```diff
@@ -1,8 +1,34 @@
 ﻿using System.Security.Claims;
+using System.Text.Json.Serialization;
 
 namespace Digdir.Domain.Dialogporten.Application.Externals.Presentation;
 
 public interface IUser
 {
     ClaimsPrincipal GetPrincipal();
+    string GetSystemUserOrg();
```

Comment:
[#suggestion: existing extension](https://github.com/Altinn/dialogporten/pull/3071/files?w=1)

☝🏼 claim principal extensions har allerede en `TryGetSystemUserOrgNumber` 👀 
Får du testa igjen med den PR'en? 
Jeg prøvde en test her og fikk et par rare navn fra navneregisteret, og null for en org 🤔